packages/api/src/controllers/asset.ts

```diff
@@ -573,6 +573,18 @@ async function createTusServer(objectStoreId: string) {
   return tusServer;
 }
 
+export const setupTestTus = async (): Promise<void> => {
+  tusServer = await createTestTusServer();
+};
+
+async function createTestTusServer() {
+  const tusTestServer = new tus.Server();
+  tusTestServer.datastore = new tus.FileStore({
+    path: "/tmp",
```

Comment:
Might wanna use `os.tmpdir()` here instead of the hardcoded `/tmp` just for compatibility with non-Unix platforms.

Can also consider using `fs.mkdtempSync` to create a folder inside `/tmp` just not to pollute devs machines too much. Could be just `/tmp/studio-tus` for example to keep everything in there. WDYT?

Found all those functions here: https://blog.mastykarz.nl/create-temp-directory-app-node-js/
But I don't think we need to bother deleting the directory after the tests are done. Seems OK to just write to the same folder every time and the dev can clean it up eventually if desired.